libs/wingsdk/src/std/map.ts

```diff
@@ -72,29 +82,37 @@ export class ImmutableMap {
 export class MutableMap extends ImmutableMap {
   /**
    * Removes all elements
+   * @macro $self$={}
    */
   public clear(): void {
-    throw new Error("Abstract");
+    throw new Error("Macro");
   }
 
   /**
    * Removes the specified element from a map.
+   *
+   * @macro delete $self$[$args$]
+   *
    * @param key The key
    * @returns true if the element was in the map
    */
   public delete(key: string): boolean {
     key;
-    throw new Error("Abstract");
+    throw new Error("Macro");
   }
 
   /**
    * Adds or updates an entry in a Map object with a specified key and a value.
+   *
+   * TODO: revisit this macro after we support indexed params
```

Comment:
What do you mean by indexed params?
+ every TODO must have a tracking issue ref

Reply:
Tracking issue added. I meant to say indexed args. `$args[0]$` type of thing